specification/metrics/pricingquantity.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Quantity
+
+The Pricing Quantity represents the volume of a given resource and/or service used or purchased based on the [Pricing Measure](#pricingmeasure). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
+
+PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeCategory](#chargecategory) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Comment:
ChargeCategory is currently named [ChargeType](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/dimensions/chargetype.md) and I'm not finding an open PR that changes that.
Also, there should be a space between those two words as the name of the column instead of the column ID.

```suggestion
PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [Charge Type](#chargetype) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Reply:
Other columns are referred to by their name so I made the above change for consistency. PricingQuantity in this paragraph is self-referential so I'm not sure whether referring to it by the name or the ID is preferred.

Reply:
We'll do a spec-wide sweep for ChargeType to Category once the PRs that reference it are in. Either is good, but it makes sense to use ChargeType in case anything comes up.

Reply:
_Updated my suggestion reflect ChargeType instead of Charge Type_

Reply:
@udam-f2 @flanakin should we switch everything back to ChargeType, AdjustmentType? Or leave it AS-IS?